RoddyCore/src/de/dkfz/roddy/core/CohortDataRuntimeServiceExtension.groovy

```diff
@@ -114,6 +118,13 @@ class CohortDataRuntimeServiceExtension {
     List<DataSet> loadCohortDatasetsWithFilter(Analysis analysis, List<String> pidFilters) {
         List<DataSet> listOfDataSets = runtimeService.getListOfPossibleDataSets(analysis)
 
+        if (checkIfListOnlyContainsSCIdentifiers(pidFilters)) {
```

Comment:
The "list" is redundant -- the signature says that is should be a list. Maybe `containsOnlySupercohortIds()`?